public/index.html

```diff
@@ -12,5 +12,39 @@
     <div id="app"></div>
     <script src="build/vendor.js"></script>
     <script src="build/app.js"></script>
+    <script>
+    if ('serviceWorker' in navigator) {
+      navigator.serviceWorker.register('./service-worker.js', {
+        scope: './'
+      })
+      .then(function(registration) {
+        if (typeof registration.update === 'function') {
+          registration.update();
```

Comment:
`navigator.serviceWorker.register` itself runs `Update` algorithm. Why this is needed? Maybe just leave everything here as simple SW registration, or you are trying to achieve some special case? Or maybe I am just missing something :-)